_pages/algorithm-development/source-control-management.md

```diff
@@ -212,3 +214,217 @@ An administrator of your GitHub organization likely needs to approve the OAuth a
+
+This will open a publish modal that will let you publish your algorithm.
+
+If you don't see any builds listed, make sure that you're pushing commits to your repository's _default_ branch. Builds are only triggered for commits pushed to the default branch.
+
+To learn more about algorithm builds, [click here]({{site.cdnurl}}{{site.baseurl}}/platform/algorithm-profiles#algorithm-builds).
+
+#### Deleting Bitbucket-Hosted Algorithms
+
+If you wish to delete an algorithm that hosts its source code in Bitbucket, have no fear: the repository will be left unharmed. If you do wish to destroy your algorithm's source code, you can follow [these instructions](https://support.atlassian.com/bitbucket-cloud/docs/delete-a-repository/).
+
+#### Managing Your Bitbucket Authorization
+
+If you want to review your Bitbucket authorization status, you can visit your user settings page. Simply scroll to the "Source Control Management" section to view any prior Bitbucket authorizations, or to connect your account:
+
+**Enterprise Users:** By default, new Algorithmia instances can only store source code internally within the Algorithmia platform. As such, you may not see Bitbucket listed within the above section until your administrator enables Bitbucket as a source host.
+{: .notice-info}
+
+#### Troubleshooting Bitbucket-Hosted Algorithms
+
+With the source for your algorithm hosted externally, there is a chance that our connection to Bitbucket could become disrupted. When this happens, it could be due to one of the following:
+
+##### Your Repository's Access Key Was Removed
+
+We depend on access keys to pull source code from your algorithm for building. A access key is simply an SSH keypair with read-only access to a specific repository. Algorithmia securely stores the private key while the public key is shared with Bitbucket.
+
+You can obtain the public key for your repository by navigating to your algorithm's settings page and clicking "View Key":
+
+You will then be presented with a modal, from which you can copy the public key.
+
+With the access key in hand, simply follow [instructions outlined here](https://support.atlassian.com/bitbucket-cloud/docs/add-access-keys/) to restore the public key to your repository.
+
+##### Your Repository Was Deleted
+
+If your algorithm's Bitbucket repository was deleted, Bitbucket does not have a mechanism for restoring. It is essential that both the repository's deploy key and webhooks remain in place, otherwise we will not be able to pull your source code, or know when you make changes.
+
+##### Your Repository's Webhook Needs to be Restored
+
+We depend on webhooks to notify us when changes have occurred to your algorithm's repository. To restore your webhook within Bitbucket, first locate the webhook URL and secret. This can be found on your algorithm's settings page by clicking 'View Details' under 'Bitbucket Webhook'.
```

Comment:
```suggestion
We depend on webhooks to notify us when changes have occurred to your algorithm's repository. To restore your webhook within Bitbucket, first locate the webhook URL. This can be found on your algorithm's settings page by clicking 'View Details' under 'Bitbucket Webhook'.
```